### PR TITLE
feat: partial separation of dynamiclink into `wasmplus`

### PR DESCRIPTION
**Pull request by da1suk8**
## Description

The dynamiclink branch was **partially** separated into wasmplus.

Separation Process
1. Merge to 60fec8ad6ab2689b268fce7566d18fef53de182d where the separation was made. Check the difference.
2. Merge to 514953c0b244dd297511f395106077aed5f2e7d7
3. Check the difference, and partially separated

Because the `cosmwasmAPI` is not exposed to the outside world, it could not be separated.
It is necessary to use a different method for separation.

When https://github.com/line/wasmvm/pull/94 is merged, update commit-hash and add CHANGELOG.

- [x] commit-hash
- [x] CHANGELOG

## How has this been tested?

```
make build
make test
```

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [x] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by da1suk8** [comment]
As for the `TestCheckLibwasmVersion` that fails in ci, it succeed if the commit of wasmvm is set appropriately instead of replace in go.mod.

**Comment by loloicci** [comment]
commit titles of [fcc1b46](https://github.com/line/wasmd/pull/22/commits/fcc1b46a44add32b31efbf5e34c1e6e6415f02e1) and [ce22554](https://github.com/line/wasmd/pull/22/commits/ce225549196b937a98fdb31ffe9c2ac298edb8e8) do not start with the kind of the commit. (for example fix, feat, ci, ...)

**Review by kokeshiM0chi — DISMISSED** [review]

**Comment by loloicci** [comment]
Can [fcc1b46](https://github.com/line/wasmd/pull/22/commits/fcc1b46a44add32b31efbf5e34c1e6e6415f02e1) 's api access the wasmplus's keeper?
It needs to get "is inactive contract" in "call callable point" which will be added in https://github.com/line/wasmvm/issues/102.

File: appplus/params/params.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package params
```
[the remark this answers is not in the document]
Comment (da1suk8) [reply appplus/params/params.go:1]: The following is added according to main branch.

**Comment by da1suk8** [comment]
> Can [fcc1b46](https://github.com/line/wasmd/pull/22/commits/fcc1b46a44add32b31efbf5e34c1e6e6415f02e1) 's api access the wasmplus's keeper?
> It needs to get "is inactive contract" in "call callable point" which will be added in https://github.com/line/wasmvm/issues/102.

Access is not possible.
Since it is a circular reference, it may need to be separated in a different way.

How about dealing with this in https://github.com/line/wasmd/issues/23？

**Comment by da1suk8** [comment]
I will consider using a different method to partial separate dynamiclink into wasmplus since they are incompatible with https://github.com/line/wasmd/pull/26.

**Comment by da1suk8** [comment]
From now on, this issue will be handled https://github.com/line/wasmd/issues/25, and the PR here will be closed.

**Pull request closed without merging**